homebrew_challenge/src/App.css

```diff
@@ -72,3 +73,20 @@ img#logo {
   color: #3c3c3b;
   font-size: 16pt;
 }
+
+.about-us h1 {
+  padding-top: 2vh;
+}
+
+.about-us p {
+  width: 72%;
```

Comment:
We've been using flexboxes elsewhere to handle responsiveness. Means we don't have to calculate percentages so much

Reply:
Oh it's just a number, wanted to avoid text from border to border. Might looks better.

Reply:
Changed it!